src/screens/wallet/wallet.tsx

```diff
@@ -11,7 +11,7 @@ import { ScreenStatusBar } from '../../components/screen-status-bar/screen-statu
 import { TokenEquityValue } from '../../components/token-equity-value/token-equity-value';
```

Comment:
Sorry, I can't put the comment in the proper place
`'ScreenStatusBar' is defined but never used`